tests/regression/test_webhook_send_admin_notification.py

```diff
@@ -0,0 +1,75 @@
+import copy
+from src.Constants import PaymentStatus
+from src.plugins.webhook import WebhookPlugin
+from src.model.reward_log import RewardLog
+
+
+# Default rewardlog
+rewardlog = RewardLog(
+    "tz1LrHNbbCLgNJZsEsTUYFvWz2THgJC8fHyX",
+    "D",
+    staking_balance=65116664916,
+    current_balance=100000000000,
+)
+rewardlog.cycle = 415
+rewardlog.ratio = 0.16080255
+rewardlog.service_fee_ratio = 0.01398283
+rewardlog.amount = 207756875
+rewardlog.service_fee_amount = 18065815
+rewardlog.service_fee_rate = 0.08
+rewardlog.payable = True
+rewardlog.skipped = False
+rewardlog.hash = "oo4Gikxyj8cMqM8xzgWxqnsxXoGwfhZqHDrLqpA6ENmMVoYUnVd"
+rewardlog.neededActivation = False
+rewardlog.paid = PaymentStatus.DONE
+
+# Second reward log with floats
+rewardlog2 = copy.deepcopy(rewardlog)
+rewardlog2.staking_balance = 123123123.0
+rewardlog2.current_balance = 1000000000.0
+rewardlog2.amount = 200066666.0
+rewardlog2.service_fee_amount = 789789789.0
+
+# Third reward log with strings
+rewardlog3 = copy.deepcopy(rewardlog)
+rewardlog3.staking_balance = "98765643"
+rewardlog2.current_balance = "1000000000"
+rewardlog3.amount = "12345678900"
+rewardlog3.service_fee_amount = "5555555"
+rewardlog3.ratio = "0.16080255"
+rewardlog3.service_fee_ratio = "0.01398283"
+rewardlog3.service_fee_rate = "0.08"
+
+cfg = {
+    "webhook": {
+        "endpoint": "https://testnet-tezos.giganode.io:443",
```

Comment:
This is not correct. Tezos RPC's do not support webhook. You need to change this to the original testing server.

Reply:
done

Reply:
will that actually work? mydomain.com/webhook.php ?

Reply:
It's only a dummy configuration which does no actual requests. 
The test only tests if the payload was generated correctly with the correct types.